config/ember-try.js

```diff
@@ -7,6 +7,105 @@ module.exports = {
       }
     },
     {
+      name: 'ember-1-12',
```

Comment:
Correct, I would have Travis test against all of them but due to Travis limiting us to 4 concurrent builds I'm not having CI run them all. By having them all defined here though a dev can easily run tests against any version via `ember try:one ember-2-3`.